marimba-web/Models/Member.cs

```diff
@@ -24,6 +24,7 @@ public class Member
         public Marimba.MemberType memberType { get; set; }
```

Comment:
I was just following the convention in the other model classes. We could keep it like this for now and have a broader discussion on get/sets in model classes

Reply:
I had a ticket to change them so that they weren't all public but we could defer it, yeah.  Would you be able to add a constructor and method to edit the fields of the Elector class?

Reply:
Done!